### DISCUSSION TITLE
aac range of bitrates extension

### DISCUSSION THREAD

**Discussion by jjisnow** [audacity/audacity#2891] [Q&A]
Dear audacity team,
    Can you please extend the bitrate encoding option range of the aac encoder from 98-320kbps to 32-320kbps? It would only require a small UI change, but would enable recording and compact exporting of large audio recordings which are what a good proportion of users might need to do

**Comment by LWinterberg** [comment]
You can use a custom FFMPEG export instead to achieve the same effect:
![image](https://user-images.githubusercontent.com/87814144/166432428-c96124d5-0898-4598-9f84-4cf48d9f3de7.png)

**Reply by jjisnow** [reply]
Thanks for that - glad that there's a workaround, but I think that it really wouldn't hurt to open up the simple UI to a wider bitrate range - if simple voice recording without attention to quality is needed, it would be nice to go as low as 32kbps
